### ISSUE TITLE
Error accessing position of SceneObject

### ISSUE THREAD

**Issue by SimoneDelPrete** [NVlabs/sionna#449]
I am trying to replicate some example I found on the documentation. In particular, I am running this code here:
```python
import os

os.environ["CUDA_VISIBLE_DEVICES"] = ""
os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"


import sionna
from sionna.rt import PlanarArray, SceneObject, Transmitter, load_scene

scene = load_scene(sionna.rt.scene.simple_street_canyon_with_cars)
scene.tx_array = PlanarArray(1, 1, 0.5, 0.5, "tr38901", "V")
scene.add(Transmitter("tx", position=[22.7, 5.6, 0.75], orientation=[0, 0, 0]))

# new_obj = SceneObject(name="new", object_id=1, scene=scene, mi_shape="square")

# print(new_obj.position)
car: SceneObject = scene.get("car_3")  # type: ignore

print(car.position)

``` 

When I run this code, I get this error when running `print(car.position)`:
```shell
2024-05-28 13:10:45.861420: F tensorflow/core/framework/tensor.cc:847] Check failed: IsAligned() ptr = 0x5654e12ee320
Aborted
``` 

I am running sionna version 0.17.0
CUDA Version: 12.5

 OS: Ubuntu 22.04.3 LTS on Windows 10 x86_64 (Linux version 5.15.146.1-microsoft-standard-WSL2)

**Comment by SebastianCa** [comment]
Hi @SimoneDelPrete,

I cannot reproduce the error on my local machine (Ubuntu 22.04). This seems related to #446 and the issue happens at the exact same point when accessing the `position` argument.

Does `python3 -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"` work now?

**Comment by SimoneDelPrete** [comment]
Hi @SebastianCa 

So the issue persists as in #446, I installed CUDA as explained in the nvidia guide, but I think that the kernel is actually not compatible with NUMA, so probably that's why tensorflow doesn't recognise GPU. In fact, if I run the command numactl I get this output:
```powershell
simonedelprete@PC-SDP:~$ numactl --show                                                                                                                          physcpubind: 0 1 2 3 4 5 6 7                                                                                                                                     No NUMA support available on this system.                                                                                                                        simonedelprete@PC-SDP:~$ numactl --hardware                                                                                                                      No NUMA available on this system
``` 

I read [here](https://github.com/microsoft/WSL/issues/5025) that is a problem related to WSL and I might need to compile my own kernel. 

As a matter of fact, I was thinking to avoid using GPU for computation (since my simulations will not be very complex)  and in fact I set os.environ["CUDA_VISIBLE_DEVICES"] = "" in order to avoid any problem with the GPU. 

Everything works fine, I tried some other examples and they work fine, only when I try to access to the position attribute it gives me that error. In case I run directly with python it gives me that error, with jupyter the kernel crashes.

**Comment by faycalaa** [comment]
Hi,

This bug will be fixed in the next release.
In the meantime, you can fix it by making the following change to [these lines](https://github.com/NVlabs/sionna/blob/8ad32bc6eca75cfe4003a6a5f035cf4a435ff84e/sionna/rt/scene_object.py#L146):
```
# Bounding box
# [3]
bbox_min = tf.cast(self._mi_shape.bbox().min, rdtype)
# [3]
bbox_max = tf.cast(self._mi_shape.bbox().max, rdtype)
```

**Comment by SimoneDelPrete** [comment]
> Hi,
> 
> This bug will be fixed in the next release. In the meantime, you can fix it by making the following change to [these lines](https://github.com/NVlabs/sionna/blob/8ad32bc6eca75cfe4003a6a5f035cf4a435ff84e/sionna/rt/scene_object.py#L146):
> 
> ```
> # Bounding box
> # [3]
> bbox_min = tf.cast(self._mi_shape.bbox().min, rdtype)
> # [3]
> bbox_max = tf.cast(self._mi_shape.bbox().max, rdtype)
> ```

Thanks, now it works!

**Comment by jhoydis** [comment]
Fixed in v0.18.
